project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.8.3
\ No newline at end of file
+sbt.version=1.9.7
```

Comment:
1.9.9 is available if you're able to upgrade this

Reply:
We decided to keep this at 1.9.7 to match platform recommendation on TC, so we agreed to keep on this repo too